_posts/2017-12-19-2017-year-in-review.md

```diff
@@ -0,0 +1,144 @@
+
+For 2018, Flink Forward will be back in [September in Berlin](https://flink-forward.org/), and in [April in San Francisco](https://sf-2018.flink-forward.org/).
+
+![Flink Forward Speakers]({{ site.baseurl }}/img/blog/speaker-logos-ff2017.png)
+
+## Features and Ecosystem
+
+### Stateful Stream Processing with dA Platform 2
+
+In September dataArtisans announced an early access program of our own SaaS product based on Apache Flink, [dA Platform 2](https://data-artisans.com/blog/da-platform-2-stateful-stream-processing-with-apache-flink-made-easier). The 'Application Manager' is an integral component of the platform and is the central coordination point between commands from the user and orchestration of cluster resources and Flink jobs.
+
+![dA Platform 2 architecture overview](https://data-artisans.com/wp-content/uploads/2017/09/application_manager_opt4@3x.png)
+
+We aim for dA Platform 2 to be generally available in the first quarter of 2018, providing a commercialy backed platform to the Flink community from the team who helped create the open source project.
+
+### Flink Ecosystem Growth
+
+Flink was added to a selection of distributions and integrations during 2017, making it easier for a wider user base to get started with Flink:
+
+-   [Official Docker image](https://hub.docker.com/r/_/flink/)
+-   [Official DC/OS and Mesos support](https://ci.apache.org/projects/flink/flink-docs-release-1.3/setup/mesos.html)
+-   [A Flink connector](https://data-artisans.com/blog/dellemc-launches-open-source-pravega-complete-apache-flink-connector) for [Pravega](http://pravega.io), Dell/EMC's streaming storage system.
+-   Uber announced AthenaX, a streaming SQL platform [powered by Apache Flink](https://data-artisans.com/blog/uber-introduces-open-source-athenax-streaming-sql-platform-apache-flink).
+
+### Feature Timeline in 2017
+
+Just in time for the end of the year, [our 1.4 release](read the full release announcement](<http://flink.apache.org/news/2017/12/12/release-1.4.0.html>) landed in mid-December culminating 5 months of work and the resolution of more than 900 issues. This is the fifth major release in the 1.x.y series.
+
+Here's a selection of major features added to Flink over the course of 2017:
+
+![Flink Release Timeline 2017]({{ site.baseurl }}/img/blog/flink-releases-2017.png)
+
+<!-- TODO: Is the number below correct? It doesn't seem that high -->
+
+If you take a look at [the resolved issues for 2017 on Jira](https://issues.apache.org/jira/browse/FLINK-4861?jql=project%20%3D%20FLINK%20AND%20issuetype%20%3D%20%22New%20Feature%22%20AND%20status%20in%20(Resolved%2C%20Closed)%20AND%20resolved%20%3E%3D%202017-01-01%20AND%20resolved%20%3C%3D%202017-12-31%20ORDER%20BY%20resolved%20ASC) you can see that the community resolved over 100 issues and feature additions.
```

Comment:
@wints I wasn't sure how to find a complete list, this seems quite small.

Reply:
@ChrisChinchilla I opened it up to include improvements and bugfixes, and with those 2 categories, we have 1,829 issues: https://issues.apache.org/jira/browse/FLINK-5016?jql=project%20%3D%20FLINK%20AND%20issuetype%20in%20(Bug%2C%20Improvement%2C%20%22New%20Feature%22)%20AND%20status%20in%20(Resolved%2C%20Closed)%20AND%20resolved%20%3E%3D%202017-01-01%20AND%20resolved%20%3C%3D%202017-12-31%20ORDER%20BY%20resolved%20ASC. That's how I generated the count last year.